fdbserver/VersionedBTree.actor.cpp

```diff
@@ -9863,7 +9883,7 @@ TEST_CASE(":/redwood/performance/set") {
 				// actor state object
 				double* pIntervalStart = &intervalStart;
 
-				commit = map(btree->commit(version), [=](Void result) {
+				commit = map(btree->commit(++version), [=](Void result) {
```

Comment:
This wasn't in the master version, what's the difference?

Reply:
I think this change went in master in a previous PR.

Reply:
This was fixed on master in #6036, I didn't port the rest of that PR to 7.0 since it wasn't really "needed".